_usage/command-line.md

```diff
@@ -35,7 +39,7 @@ Where:
 
 ### Connecting with SSMS
 
-The SQL Server Management Studio is an additional option to talk to Babelfish. 
+The SQL Server Management Studio is an additional option to talk to Babelfish.
 
 In the following procedure, you connect to your Babelfish database by
 using Microsoft SQL Server Management Studio (SSMS). You can use the
```

Comment:
Use product names consistently. There is 'SQL Server Management Studio', with and without 'Microsoft'. Sometimes the SSMS abbrev is used. Be consistent, and introduce an abbreviation at the first use of the name.

Reply:
Ok. Variatio delectat, but not everybody.